docs/workloads/azure/data/etl_pipelines/data_quality_azure.md

```diff
@@ -35,3 +34,89 @@ The design of the Data Quality processing is outlined in the following diagram.
+
+```bash
+datastacks dq --help
+datastacks dq --config-path "ingest/Ingest_AzureSql_Example/data_quality/ingest_dq.json" --container config
+```
+
+`config-path` is a path to a JSON config inside the Azure Blob container, `container` is name of the container. The storage account used will be based upon the `BLOB_ACCOUNT` environment variable.
+
+## JSON Configuration File for Great Expectations
+
+This section describes the structure of the JSON configuration file used in our system.
+The configuration is defined using Python's Pydantic library for data validation.
+
+Here is the description of the main elements:
+
+1. `gx_directory_path`: Path to the Great Expectations metadata store.
+2. `dataset_name`: Name of the dataset that is being processed.
+3. `datasource_config`: List of datasource configurations where each configuration contains the following fields:
+    1. `datasource_name`: Name of the data asset, e.g., table or file name.
+    2. `datasource_type`: Source system type that Spark can read from, e.g. table, parquet, csv, delta.
+    3. `data_location`: Location of the given data asset. It can either be a path to the data file
+    or a fully qualified table name, depending on the data source. Expectations for each scenario:
+       * **File Path**: If the data is stored in a file, like a Parquet file on ADLS, you should
+       provide the complete path to the file. Examples:
+          * `"abfss://raw@accountname.dfs.core.windows.net/myfolder/mysubfolder/myfile.parquet"`,
+          * `"abfss://raw@accountname.dfs.core.windows.net/myfolder/mysubfolder/*"`,
+          * `"abfss://staging@{ADLS_ACCOUNT}.dfs.core.windows.net/myfolder/mysubfolder/*"`.
+       * **Table Name**: For tables with metadata managed by a data catalog, you should provide
+       the database schema and the table name. For example, `staging.table_name`.
+    4. `expectation_suite_name`: Name of the expectation suite associated with this data source.
+    5. `validation_config`: A list of validation configurations where each configuration contains the following fields:
+        1. `column_name`: Name of the validated column.
+        2. `expectations`: List of expectations where each expectation has the following fields:
+        3. `expectation_type`: Name of the Great Expectations expectation class to use.
+        4. `expectation_kwargs`: The keyword arguments to pass to the expectation class.
+
+### Example
+
+Here's a minimal example of a configuration file:
+
+```json
+{
+    "gx_directory_path": "/dbfs/great_expectations/",
+    "dataset_name": "movies_dataset",
+    "dq_output_path": "abfss://staging@{ADLS_ACCOUNT}.dfs.core.windows.net/Ingest_AzureSql_Example/",
+    "datasource_config": [
+        {
+            "datasource_name": "movies_metadata",
+            "datasource_type": "table",
+            "data_location": "staging.movies_metadata",
```

Comment:
Is this a valid example? Would you not need to specify the full path, i.e `abfss://raw@{ADLS_ACCOUNT}.dfs.core.windows.net/pipeline_name/schema.table/v#/*/*/*/*`

Reply:
I believe this syntax is correct if was using a `table` datasource_type - however we're not currently saving data as tables (@lorraine-houston-ensono could you confirm that?) so agree it'd be better to show an similar to the ones we have with parquet or delta.